native/src/routes/__tests__/SprungbrettOfferContainer.spec.tsx

```diff
@@ -61,80 +57,44 @@ describe('SprungbrettOfferContainer', () => {
   const mockStore = configureMockStore()
   const store = mockStore(state)
 
-  const mockUseLoadFromEndpointOnce = mock => {
-    mocked(useLoadFromEndpoint).mockImplementationOnce(mock)
-  }
-
   beforeEach(() => {
     jest.clearAllMocks()
   })
 
   it('should display offers without a Loading spinner', () => {
```

Comment:
Sorry, I was wrong. It is possible that useLoadFromEndpoint returns data (or an error) and loading true as the data/error is not instantly set to null when refreshing. I think this is fine to allow pull to refresh behavior and still show previously loaded data. So maybe it would be best to readd those tests.